### PR TITLE
Add rule regarding ball play on the nearest neutral spot

### PR DESCRIPTION
**Pull request by marcodankel**
### Please describe your change in one or two sentences

Add a rule regarding ball play: Robots must be able to approach and play a ball when it's placed on the nearest neutral spot. Also, when left alone, robots must be able to maneuver the ball to the opponents side of the field.

### Please explain why do you think this change should be in the rules

This new rule shall remove the ability of a goalie to simply block its own goal without leaving the goalie area.

### CODE REVIEW

File: rules.adoc [hunk 1]
```diff
@@ -199,9 +199,19 @@ imparts dynamic back spin on the ball to keep the ball on its surface.
...
 walls. If a robot moves the ball outside of the field (that is, beyond the walls
-or above their height), it is deemed damaged. (<<damaged-robots>>)++}
+or above their height), it is deemed damaged. (<<damaged-robots>>)
+
+{~~A robot must touch the ball that is placed no further than 20 cm from any point
+on its convex hull within 10 seconds. If a robot does not do so within the time
+limit, it is deemed to be damaged. ~>
+Any robot must approach and touch the ball when it is placed on the nearest
+neutral spot before lack of progress is called. When on its own side of the field,
+any robot must be able to move the ball from the nearest neutral spot to the
```
Comment (Adman) [inline rules.adoc:211]: What if the neutral spot is on the opponent's side?

File: rules.adoc [hunk 2]
```diff
@@ -199,9 +199,19 @@ imparts dynamic back spin on the ball to keep the ball on its surface.
...
+opponent's side of the field. If a robot does not do these things, it is deemed to be damaged.
```
Comment (Adman) [inline rules.adoc:212]: Which robot is deemed to be damaged? All on the field? The nearest one?

Comment (marcodankel) [reply rules.adoc]: That's going to be the one that is not able to play the ball on the nearest neutral spot. 🙂

File: rules.adoc [hunk 3]
```diff
@@ -204,6 +204,16 @@ The ball needs to stay within the bounds of the field, as defined by the
 walls. If a robot moves the ball outside of the field (that is, beyond the walls
 or above their height), it is deemed damaged. (<<damaged-robots>>)
 
+{~~A robot must touch the ball that is placed no further than 20 cm from any point
+on its convex hull within 10 seconds. If a robot does not do so within the time
+limit, it is deemed to be damaged.
+~>Any robot must approach and touch the ball when it is placed on the nearest
+neutral spot before lack of progress is called. When on its own side of the field,
+any robot must be able to move the ball from the nearest neutral spot to the
+opponent's side of the field. If a robot does not do these things, it is deemed to be damaged.
```
Comment (mrshu) [inline rules.adoc:213]:
```suggestion
opponent's side of the field. If a specific robot does act this way, it is deemed to be damaged.
```

**Review by mrshu — APPROVED** [review]
I've added a small clarification to resolve the ambiguity pointed out by @Adman

File: rules.adoc (hunk 3 above)
Comment (Adman) [reply rules.adoc:213]: `Does not` act or `does act`?

**Pull request merged**